app/controllers/admin_controller.rb

```diff
@@ -17,6 +17,22 @@ def holdings
 
   private
 
+  def mediated_pages
+    if params[:expired]
+      expired_mediated_pages
+    else
+      active_mediated_pages
+    end
```

Comment:
:+1: I think I have addressed all three of these points.  Let me know if I'm missing something or if there's a better way to handle it.